app/models/affordability_analyst.rb

```diff
@@ -0,0 +1,15 @@
+class AffordabilityAnalyst
+
+
+  def find_closest_rent_neighborhoods(user_max_rent)
+    max_avg_rent = user_max_rent * 1.2
+    neighborhoods = Neighborhood.where('rent <= ?', max_avg_rent).order(rent: :desc).limit(3)
```

Comment:
I'm curious how we decided to do 120% of their suggested rent. It might be good to test over time to make sure the top three neighborhood averages are at least reaching below the user-provided rent. What do you think?